CorsixTH/Lua/entities/patient.lua

```diff
@@ -485,24 +527,74 @@ function Patient:tapFoot()
   end
 end
 
-function Patient:goHome(cured)
+-- Increments "cures" statistics
+function Patient:countCured()
```

Comment:
The name implies that you just count something, but nothing is returned and it's an increment. Maybe something like incrementCuredCount?